internal/blocktx/store/postgresql/get_stale_chain.go

```diff
@@ -0,0 +1,78 @@
+package postgresql
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/bitcoin-sv/arc/internal/blocktx/blocktx_api"
+)
+
+func (p *PostgreSQL) GetStaleChainBackFromHash(ctx context.Context, hash []byte) ([]*blocktx_api.Block, error) {
```

Comment:
Could you add here a comment which explains what this function does?
It looks not easy to understand.

Reply:
I tried to explain it a bit in the comment above the function. Let me know if that makes sense 🙏

Reply:
I added one more comment explaining the query a bit more.

But a better explanation of recursive queries can be found here: https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-recursive-query/